dlrover/python/elastic_agent/torch/training.py

```diff
@@ -1173,7 +1173,7 @@ def _report_failure_to_master(self, failures: Dict[int, ProcessFailure]):
     def _restart_workers(self, worker_group: WorkerGroup):
         self._restart_count += 1
         self._remaining_restarts -= 1
-        # Relase the shared memory lock before starting workers.
+        # Release the shared memory lock before starting workers.
         AsyncCheckpointSaver.reset()
```

Comment:
The reset method has released the shared lock.
https://github.com/intelligent-machine-learning/dlrover/blob/d3b4e449369e3efc9134a7757ef4822dbe60deac/dlrover/python/elastic_agent/torch/ckpt_saver.py#L854-L860